check-diff.sh

```diff
@@ -383,9 +383,9 @@ function download {
 
 function can_generate_coverage_clover {
 	if [ -e .coveralls.yml ] && [ -e composer.json ] && check_should_execute 'coverage'; then
-		if min_php_version "5.5.0" && cat composer.json | grep -Eq '"satooshi/php-coveralls"\s*:\s*"dev-master"'; then
+		if min_php_version "5.5.0" && cat composer.json | grep -Eq '"satooshi/php-coveralls"\s*:\s*"([(\^|~)]*(2|1.1)[0-9.]*|dev-master)"'; then
```

Comment:
Isn't it `php-coveralls/php-coveralls` now? See https://github.com/php-coveralls/php-coveralls